hooks/post-command

```diff
@@ -100,7 +100,7 @@ if [[ -n "${BUILDKITE_PLUGIN_TRIVY_SKIP_DIRS:-}" ]] ; then
   echo "skipping directories '$BUILDKITE_PLUGIN_TRIVY_SKIP_DIRS' from scan "
 fi
 
-echo "--- scanning filesystem"
+echo "+++ scanning filesystem"
```

Comment:
If you want this to be expanded by default. Let's instead make this a title instead of just replacing the characters for the `+` symbol. e.g.
```
echo "Scanning Filesystem..."
```
Or something of the sort.

Reply:
@JAORMX What do you mean? This is the recommend way by [buildkite](https://buildkite.com/docs/pipelines/managing-log-output#collapsing-output)

Reply:
I didn't know. Having a reference to the docs in the PR description/commit would be a helpful reference.